client/src/components/ProfileEdit/index.js

```diff
@@ -0,0 +1,303 @@
+import React, { useState } from 'react';
+import { 
+	Grid,
+	Typography,
+	TextField,
+	MenuItem,
+	Button,
+	useMediaQuery,
+	useTheme
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+import { months, genders, days, years } from './data';
+
+const useStyles = makeStyles(theme => ({
+	label: { 
+		fontWeight: 'bold',
+		letterSpacing: '0.7px',
+		cursor: 'pointer',
+		padding: '1.5em',
+		display: 'block',
+		textAlign: 'right'
+	},
+	mainContainer: { 
+		margin: '2em auto',
+		maxWidth: '900px'
+	},
+	input: { 
+		marginBottom: '1em',
+	},
+	select: { 
+		width: "100%",
+	},
+	button: { 
+		textTransform: 'none',
+		fontWeight: 'bold'
+	},
+	bigRedButton: { 
+		...theme.buttons.bigRedButton,
+		marginTop: '2.5em'
+	}
+}));
+
+const InputText = function({ id, placeholder, children, handleChange }) {
```

Comment:
We may want to align between this and #28 because Michael S added an Input component and a form handling function that could be helpful here